src/Components/SmartComponents/SystemCves/SystemCves.js

```diff
@@ -155,6 +155,13 @@ class SystemCves extends Component {
             openedCves = new Set();
         }
 
+        if (!Array.isArray(cveNames)) {
+            const visibleCves = this.props.cveList.data;
+            const visibleOpenedCves = new Set(visibleCves.filter(cve => openedCves.has(cve.id)));
```

Comment:
@leSamo  what is the purpose of using `new Set` here?

Reply:
fixed